lib/active_model/serializer/adapter.rb

```diff
@@ -1,26 +1,28 @@
 module ActiveModel
   class Serializer
-    class Adapter
+    module Adapter
       UnknownAdapterError = Class.new(ArgumentError)
       ADAPTER_MAP = {}
       private_constant :ADAPTER_MAP if defined?(private_constant)
       require 'active_model/serializer/adapter/fragment_cache'
       require 'active_model/serializer/adapter/cached_serializer'
 
-      def self.create(resource, options = {})
-        override = options.delete(:adapter)
-        klass = override ? adapter_class(override) : ActiveModel::Serializer.adapter
-        klass.new(resource, options)
-      end
+      class << self # All methods are class functions
+        def new(*args)
+          fail ArgumentError, [args, caller[0]].inspect
+        end
```

Comment:
Just in case.. but perhaps I could write a better message..  is really meant to be a helpful deprecation warning